sdks/intro.md

```diff
@@ -4,12 +4,25 @@ sidebar_position: 1
 
 # Introduction
 
-## What is iCure ?
+{{#medtech}}
+## What is iCure for Medtech ?
+{{/medtech}}
+{{#ehrlite}}
+## What is iCure for EHR Lite ?
+{{/ehrlite}}
 
 *iCure* is a trustable service ensuring the privacy of your patient medical data thanks to [end-to-end encryption](explanations/encryption/introduction.mdx), 
-meaning your users are the only ones who decide who can access their data. 
+meaning your users are the only ones who decide who can access their data.
+
+{{#medtech}}
+iCure lets you focus on the core features that are going to make your medical device solution successful: the user experience of your app, your algorithm, … while offloading to us the management of your data store, the monitoring of your solution, the release management of the apis, the security of the data.
```

Comment:
```suggestion
iCure lets you focus on the core features that are going to make your medical device solution successful: the user experience of your app, your algorithm, … while offloading to us the management of your data store, the monitoring of your solution, the release management of the apis, and the security of the data.
```

Also, I don't reallhy like the ... I would put a third element instead (like your priorities?) but I have no suggestion about it